api/middleware/authRequired.js

```diff
@@ -1,7 +1,7 @@
 const createError = require('http-errors');
 const OktaJwtVerifier = require('@okta/jwt-verifier');
 const { oktaVerifierConfig, okta } = require('../../config/okta');
-const Profiles = require('../profile/profileModel');
+const Profiles = require('../provider/providerModel');
```

Comment:
is this staying as Profiles? otherwise this code looks clean and good!

Reply:
Good catch!  Should probably change the const to Providers, as well as all of its uses elsewhere in the code

Reply:
I agree, I think it should be changed to Providers to avoid that confusion. If you right click and select "Find All References" it will show everywhere that specific variable implementation is used in the project.

Once you verify that, you can again right click it, and then select "Rename Symbol", type the new variable name, and save. All the references will be changed to match that new name.